### ISSUE TITLE
NuGet package does not install properly (v.0.85.0)

### ISSUE THREAD

**Issue by Rodney-Thompson** [EvotecIT/PSWriteColor#3]
When manually downloading and installing the package, it installs, but then can not be imported (or seen with Get-Module -ListAvailable). This appears to be as the Module Version in the Manifest does not match the folder name. Error message:
`import-module : The ModuleVersion key in module manifest 'C:\Program
Files\WindowsPowerShell\Modules\PSWriteColor\0.85.0\PSWriteColor.psd1' specifies module version '0.85' which does not
match its version folder name at 'C:\Program Files\WindowsPowerShell\Modules\PSWriteColor\0.85.0'. Change the value of
the ModuleVersion key to match the version folder name.
At line:1 char:1
+ import-module 'C:\Program Files\WindowsPowerShell\Modules\PSWriteColo ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (C:\Program File...WriteColor.psd1:String) [Import-Module], InvalidOper
   ationException
    + FullyQualifiedErrorId : Modules_InvalidModuleManifestVersion,Microsoft.PowerShell.Commands.ImportModuleCommand`

**Comment by PrzemyslawKlys** [comment]
Hi,

Can you explain how you're actually using this?

![image](https://user-images.githubusercontent.com/15063294/53562683-4efbe980-3b52-11e9-8bd0-a45223f53ecf.png)

I can't reproduce it, but maybe you're installing it in another way.

Przemek

**Comment by Rodney-Thompson** [comment]
Hi Przemek,

As the server (2019) has no connection to the internet, I had to do everything offline with downloaded packages. As the server had no NuGet, I copied that across from my Win 10 laptop, created a local psrepository then installed using the downloaded nugget packages:
Register-PSRepository -Name localPSGallery -SourceLocation C:\ PSGalleryModules -InstallationPolicy Trusted
Install-Module pswindocumentation – this installed all the dependencies once I had them in the folder:
[cid:image001.png@01D4D019.ACF47A10]

Install-Module successfully installed all of them (I wasn’t using -verbose) but Get-Module -ListAvailable pswritecolor failed to show the module. After running Install-Module pswritecolor -Force -Verbose, I found the message that he manifest and folder names were mismatched.

Cheers,

Rodney.
